src/style.css

```
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+}
+
+.search {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.search__small {
+  height: 42px;
+}
+
+.input {
+  height: 100%;
+  width: 100%;
+  font-family: inherit;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 91, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19px;
+}
+
+.input__small {
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  padding-left: 33px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
```

remove it to block .input

```suggestion
  background-image: url(images/Search.svg);
  background-repeat: no-repeat;
```